api_tests/base/test_auth.py

```diff
@@ -457,3 +460,67 @@ def test_user_email_scope_cannot_read_other_email(self, mock_user_info):
         assert_equal(res.status_code, 200)
         assert_not_in('email', res.json['data']['attributes'])
         assert_not_in(self.user2.username, res.json)
+
+
+@pytest.mark.django_db
+class TestCSRFValidation:
+
+    @pytest.fixture
+    def user(self):
+        return AuthUserFactory()
```

Comment:
Minor: Since we're not using basic auth, you can use `UserFactory` instead of `AuthUserFactory`, which is slower to create because it has to set a password.